docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use official Java base image
+FROM java:7-jre
+MAINTAINER Dennis Pallett, dennis.pallett@topicus.nl
+
+# Make the log directory a volume so it's available to other containers
+VOLUME ["/logs"]
+
+# set entrypoint to shell script
+ENTRYPOINT ["/docker-entrypoint.sh"]
+
+# Add MSSQL2MonetDB jar file
+ENV TOOL_VERSION 0.5
+COPY mssql2monetdb-$TOOL_VERSION-jar-with-dependencies.jar /mssql2monetdb-$TOOL_VERSION.jar
```

Comment:
Where does the jar comes from? Would be better to download it from github files.